sympy/external/tests/test_codegen.py

```diff
@@ -115,6 +118,8 @@
 
 def try_run(commands):
     """Run a series of commands and only return True if all ran fine."""
+    if pyodide_js:
```

Comment:
Can we set this to SKIP instead of just doing nothing?

Reply:
TL;DR: it's equivalent to SKIP
```
sympy/external/tests/test_codegen.py[9] sssss....                           [OK]
```
Detail:
If `try_run` returns `False`, then `compiled` is `False`
https://github.com/sympy/sympy/blob/587275372c1055a37a0db1f317461c8295b36fca/sympy/external/tests/test_codegen.py#L189
So this assertion fails
https://github.com/sympy/sympy/blob/587275372c1055a37a0db1f317461c8295b36fca/sympy/external/tests/test_codegen.py#L217-L218
And `is_feasible` returns `False`
https://github.com/sympy/sympy/blob/587275372c1055a37a0db1f317461c8295b36fca/sympy/external/tests/test_codegen.py#L250-L255
Then this result is stored in `invalid_lang_compilers`
https://github.com/sympy/sympy/blob/587275372c1055a37a0db1f317461c8295b36fca/sympy/external/tests/test_codegen.py#L261-L264
Finally leads to SKIP
https://github.com/sympy/sympy/blob/587275372c1055a37a0db1f317461c8295b36fca/sympy/external/tests/test_codegen.py#L268-L270